### PR TITLE
feat(Task-10): As a User, I want to be to see the weather reports of 5 days

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: src/app/shared/components/forecast-card/forecastCard.component.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<md-card class="fill-content">
+    <md-card-header>
+        <md-card-header-text>
+            <div ng-show="$ctrl.currentForecast" class="flex-row justify-space-around">
+                <span class="md-title">Sunrise: {{ $ctrl.currentForecast.sys.sunrise * 1000 | date:'HH:mm:ss' }}</span>
+                <span class="md-title">
+                    <img ng-src="https://openweathermap.org/img/w/{{$ctrl.currentForecast.weather[0].icon}}.png"
```
Comment (eloydotco) [inline src/app/shared/components/forecast-card/forecastCard.component.html]: I think `$ctrl.currentForecast.weather[0].icon` can have one-time binding (::), right?

File: src/app/shared/components/forecast-card/forecastCard.component.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                        class="md-avatar icon-size" />
+                    {{ $ctrl.currentForecast.name }} ({{ $ctrl.currentForecast.sys.country }})
```
Comment (eloydotco) [inline src/app/shared/components/forecast-card/forecastCard.component.html]: Also here

File: src/app/shared/components/forecast-card/forecastCard.component.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        </md-card-header-text>
+    </md-card-header>
+    <md-card-content ng-class="{'make-column': $ctrl.currentForecast}">
+        <div ng-show="$ctrl.currentForecast" class="flex-row today-content flex-wrap">
+            <progress-bar icon-to-display="cloud" progress="$ctrl.currentForecast.clouds.all" color="#878582">
+            </progress-bar>
+            <progress-bar icon-to-display="opacity" progress="$ctrl.currentForecast.main.humidity" color="#a9dcf4">
+            </progress-bar>
+            <span class="flex-25">
+                <md-icon class="material-icons temp">whatshot</md-icon>{{$ctrl.currentForecast.main.temp}}ºC
```
Comment (eloydotco) [inline src/app/shared/components/forecast-card/forecastCard.component.html]: Also here??

File: src/app/shared/components/forecast-card/forecastCard.component.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <span class="pressure flex-25">
+                <md-icon class="material-icons">vertical_align_bottom</md-icon>{{$ctrl.currentForecast.main.pressure}}
+                hPa
+            </span>
+        </div>
+        <md-list ng-class="{'justify-center': $ctrl.currentForecast}" class="md-dense flex-row flex-wrap">
+            <md-list-item class="justify-center"
+                ng-class="{'flex-25': $ctrl.currentForecast, 'flex-100': !$ctrl.currentForecast}"
+                ng-repeat="forecast in $ctrl.forecastData track by $index">
+                <img ng-src="https://openweathermap.org/img/w/{{forecast.weather[0].icon}}.png" class="md-avatar" />
```
Comment (eloydotco) [inline src/app/shared/components/forecast-card/forecastCard.component.html:44]: and also here :)

File: src/app/shared/components/progress-bar/progressBar.component.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<md-icon ng-show="$ctrl.iconToDisplay" class="material-icons">{{ $ctrl.iconToDisplay }}</md-icon>
```
Comment (eloydotco) [inline src/app/shared/components/progress-bar/progressBar.component.html]: Can be used here an one-time binding??

**Review by eloydotco — APPROVED** [review]

File: src/app/shared/components/progress-bar/progressBar.controller.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+export default class ProgressBarController {
+    constructor() {
+
+    }
+
+    getPercentage() {
+        return this.progress + "%";
```
Comment (diegopf) [inline src/app/shared/components/progress-bar/progressBar.controller.js]: Better use a string template here

File: src/app/app.module.js [hunk 7]
```diff
@@ -1,16 +1,16 @@
-//Angular and Angularmd
+//Angular, Angularmd ns uiRoutes
```
Comment (diegopf) [inline src/app/app.module.js]: ns?

File: src/app/search-page/searchPage.controller.js [hunk 8]
```diff
@@ -22,7 +23,41 @@ export default class SearchPageController {
...
+            .catch((error) => {
+                console.log(error); //TODO on next tickets
+            });
+    }
+
+    filterFiveDayData(data) {
+        this.todayForecast = [];
+        this.fiveDayForecast = new Map();
+
+        data.data.list.forEach((hourlyForecast) => {
```
Comment (diegopf) [inline src/app/search-page/searchPage.controller.js]: You should check data is coming and the format is the expected

**Review by diegopf — CHANGES_REQUESTED** [review]

File: src/app/shared/components/progress-bar/progressBar.component.scss [hunk 9]
```diff
@@ -0,0 +1,36 @@
+progress-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+    md-icon {
+        margin-right: 5px;
+        margin-left: 5px;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/shared/components/progress-bar/progressBar.component.scss]: you can do this in one line: `margin: 0 5px;`

File: src/app/search-page/searchPage.component.scss [hunk 10]
```diff
@@ -22,6 +22,26 @@ search-page {
         }
     }
 
+    forecast-card {
+        padding-left: 10px;
+        padding-top: 10px;
+        padding-bottom: 10px;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/search-page/searchPage.component.scss]: Yo can do this in one line: `padding: 10px 0 10px 10px;`

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**